src/clj/stripe/transfer.clj

```diff
@@ -69,8 +69,8 @@
 
 
 (s/def ::create-params
-  (-> (s/keys :req-un [::amount ::currency ::destination]
-              :opt-un [::source_transaction ::transfer_group])
+  (-> (s/keys :opt-un [::amount ::currency ::destination
```

Comment:
Why is `amount` a param at all, given that it's part of the required parameters for the create function?

Reply:
Right! It is redundant. Fixing...